src/js/backend.ts

```diff
@@ -1,14 +1,17 @@
 import { BrowserBackend } from "@sentry/browser/dist/backend";
-import { BaseBackend, NoopTransport } from "@sentry/core";
+import { BaseBackend, getCurrentHub, NoopTransport } from "@sentry/core";
 import { BrowserOptions, Transports } from "@sentry/react";
 import { Event, EventHint, Severity, Transport } from "@sentry/types";
+import { timestampInSeconds } from "@sentry/utils";
 // @ts-ignore LogBox introduced in RN 0.63
 import { Alert, LogBox, YellowBox } from "react-native";
 
 import { ReactNativeOptions } from "./options";
 import { NativeTransport } from "./transports/native";
 import { NATIVE } from "./wrapper";
 
+const jsStartedTimestampSeconds = timestampInSeconds();
```

Comment:
nativeStartedTime is in order of millis, this one in order of seconds, can we get millis precision?

Reply:
The timestamps in @sentry/tracing use seconds with 3 decimals for millis, `timestampInSeconds` returns such